Kudu.Core/SourceControl/Git/LibGit2SharpRepository.cs

```diff
@@ -239,9 +240,15 @@ public void FetchWithoutConflict(string remoteUrl, string branchName)
             }
             catch (LibGit2SharpException exception)
             {
-                if (exception.Message.Equals("Unsupported URL protocol"))
+                tracer.TraceWarning("LibGit2SharpRepository fetch failed with {0}", exception);
+
+                // LibGit2Sharp doesn't support SSH yet. Use GitExeRepository
+                // LibGit2Sharp only supports smart Http protocol
+                if (exception.Message.Equals("Unsupported URL protocol") ||
```

Comment:
The explicit retry on specific error is a good start.  We could revisit as we hear more from customers.   There is a workaround to disable libgit.   So users will not be blocked.